### PR TITLE
Adding Peri and Brett from logging team to loki OWNERS files

### PR DESCRIPTION
**Pull request by ewolinetz**
They are contributing to the Loki on Observatorium effort, adding them here in case there are additional steps they need to add here.

cc @ingvagabund

### CODE REVIEW

File: ci-operator/step-registry/gather/OWNERS
Comment (wking) [inline ci-operator/step-registry/gather/OWNERS:9]: Can we prune the rest of this file?  Most of them will still have approver powers via membership in the generic `ci-operator/step-registry/gather/OWNERS`, but I expect they lack specific understanding around how Loki gathering works.  I know that I'm personally lacking special Loki-gathering insight.

Comment (wking) [reply ci-operator/step-registry/gather/OWNERS:9]: Oops, you're actually touching `gather/OWNERS` here.  I think you mean to be touching `gather/loki/OWNERS`.

File: ci-operator/step-registry/openshift/e2e/aws/loki/OWNERS
Comment (wking) [inline ci-operator/step-registry/openshift/e2e/aws/loki/OWNERS:2]: And similar changed to `openshift/e2e/gcp/loki`?  I expect `openshift/upgrade/gcp/loki` should be @vrutkovs and me and any other interested parties, but I can handle that in follow-up work after this lands.

File: ci-operator/step-registry/gather/OWNERS
Comment (ewolinetz) [reply ci-operator/step-registry/gather/OWNERS:9]: ack, i'll fix this...

File: ci-operator/step-registry/openshift/OWNERS
Comment (wking) [inline ci-operator/step-registry/openshift/OWNERS:13]: This isn't Loki-scoped either.

**Review by vrutkovs — CHANGES_REQUESTED** [review]
Agree with Trevor - lets remove Loki folks from `ci-operator/step-registry/openshift/OWNERS`

File: ci-operator/step-registry/openshift/OWNERS
Comment (ewolinetz) [reply ci-operator/step-registry/openshift/OWNERS:13]: odd, i don't recall adding them here specifically... could this be a result of `make registry-metadata` ?

**Comment by ewolinetz** [comment]
@wking @vikaslaad if i remove them from `ci-operator/step-registry/openshift/OWNERS` the `prow/step-registry-metadata` job fails

**Comment by vrutkovs** [comment]
I think `make update` would fix that, `metadata.json` needs to be regenerated

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Comment by ewolinetz** [comment]
@ingvagabund had to rebase... ptal again

**Pull request merged**
